src/messaging/ExchangeContext.h

```diff
@@ -166,6 +166,12 @@ class DLL_EXPORT ExchangeContext : public ReliableMessageContext, public Referen
 
     void SetResponseTimeout(Timeout timeout);
 
+    System::Clock::Milliseconds32 GetAckTimeout();
```

Comment:
Please document that this is the amount of time we expect to pass before the exchange gives up waiting for an ack and times out.

Reply:
Done, thanks